pvlib/bifacial/infinite_sheds.py

```diff
@@ -401,7 +400,8 @@ def _shaded_fraction(solar_zenith, solar_azimuth, surface_tilt,
 
 def get_irradiance_poa(surface_tilt, surface_azimuth, solar_zenith,
                        solar_azimuth, gcr, height, pitch, ghi, dhi, dni,
-                       albedo, iam=1.0, npoints=100):
+                       albedo, dni_extra=None, model='isotropic', iam=1.0,
```

Comment:
Nit: I would switch the order of `dni_extra` and `model` in the argument list, so that `dni_extra` follows model in the Parameter list. Not important.